### PR TITLE
Mtb to open bridge text

### PR DESCRIPTION
**Pull request by Erin-Mounts**

### CODE REVIEW

File: src/Faq.tsx [hunk 1]
```diff
@@ -180,47 +184,11 @@ const Faq: FunctionComponent<{ onJoin: () => void }> = ({ onJoin }) => {
...
-                    <p>
-                        <strong>Washington University, St. Louis</strong> uses Mobile Toolbox in a cohort of
-                        brain-healthy individuals who originally participated in the Human
-                        Connectome Project. Launched March 2022.
-                    </p>
-                    <p>
-                        Sage Bionetworks uses Mobile Toolbox in 4 sub-populations of their
-                        existing mPower 2.0 digital health studies in Parkinson’s Disease
-                        populations.
+                        Lorem etc. etc.
```
Comment (hallieswan) [inline src/Faq.tsx:193]: What do you think about adding a `TODO` comment here, so it's easily findable for update in the future?

File: src/components/home/About.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ const About: FunctionComponent<{}> = () => {
         <Grid item xs={12} lg={6}>
           <Box>
             <Typography variant="h3" color={theme.palette.text.secondary} mb={10}>
-            Mobile Toolbox comprises a team of clinical colleagues, cognition researchers, measurement scientists and technical experts with proven success in multiple large-scale validation and development projects. Co-Principal Investigators Drs. Gershon and Mangravite are at Northwestern University and Sage Bionetworks respectively. Dr. Martin Silwinski of Penn State University leads the integration of the Mobile Toolbox collaboration with Mobile Monitoring of Cognitive Change (M2C2).
+           Lorem ipsum and you know the rest.
```
Comment (hallieswan) [inline src/components/home/About.tsx]: Maybe a `TODO` here as well?

**Review by hallieswan — APPROVED** [review]
Looks good to me!

File: src/Faq.tsx (hunk 1 above)
Comment (Erin-Mounts) [reply src/Faq.tsx:193]: :+1:

File: src/components/home/About.tsx (hunk 2 above)
Comment (Erin-Mounts) [reply src/components/home/About.tsx]: This one is weird--not sure it's actually used anywhere on the site, but the original text here was the same as the body text in the About Us section in AboutUs.tsx so I copied the new body text from there just to be sure.

Comment (syoung-smallwisdom) [inline src/components/home/About.tsx]: here on Gilligan's Island!

File: src/components/home/ExperiencesTextInfo.tsx [hunk 3]
```diff
@@ -145,7 +145,7 @@ export const webTabInfo: TabInfo[] = [
     body: (
       <>
         <p>
-          Make the Mobile toolbox app your own by adding custom messages, logo,
+          Make the Open Bridge app your own by adding custom messages, logo,
           and color scheme. Review custom changes in real-time
```
Comment (syoung-smallwisdom) [inline src/components/home/ExperiencesTextInfo.tsx]:
```suggestion
          and color scheme. Review custom changes in real-time.
```

File: src/components/home/Home.tsx [hunk 4]
```diff
@@ -20,14 +20,14 @@ const Home: FunctionComponent<{onJoin: () => void}> = ({onJoin}) => {
...
-                <Typography variant="h1">Mobile Toolbox</Typography>
+                <Typography variant="h1">Open Bridge</Typography>
                 <Typography
                   variant="largeBodyCopy"
                   maxWidth="800px"
                   component={'p'}
                   mb={0}
                   mt={10}>
-                  Mobile Toolbox delivers digital tests of cognition to research study participants through a smartphone app, allowing researchers to conduct their studies remotely. Our mission includes the creation of a library of well-validated and easy-to-use remote health measures that are widely available to the entire research community particularly those studying cognitive function and aging.{' '}
+                  Open Bridge delivers digital tests of cognition to research study participants through a smartphone app, allowing researchers to conduct their studies remotely. Our mission includes the creation of a library of well-validated and easy-to-use remote health measures that are widely available to the entire research community particularly those studying cognitive function and aging.{' '}
```
Comment (syoung-smallwisdom) [inline src/components/home/Home.tsx]:
```suggestion
                  Open Bridge delivers digital tests of cognition to research study participants through a smartphone app, allowing researchers to conduct their studies remotely. Our mission includes the creation of a library of well-validated and easy-to-use remote health measures that are widely available to the entire research community -- particularly those studying cognitive function and aging.{' '}
```

I think? Or a comma, maybe?

File: src/components/home/Partners.tsx [hunk 5]
```diff
@@ -15,73 +15,24 @@ const Partners: FunctionComponent<{}> = () => {
...
-        </Grid>
         <Grid item xs={12} lg={4} textAlign="center">
           <SageBio_Logo />
         </Grid>
-        <Grid item xs={12} lg={4} textAlign="center">
-          <img src={pensate} />
-        </Grid>
         <Grid item xs={12} textAlign="center">
           <Button color="primary" variant="contained" href="/about-us">
             View Our Contributors
```
Comment (syoung-smallwisdom) [inline src/components/home/Partners.tsx:30]: WashU?

**Review by syoung-smallwisdom — APPROVED** [review]
Looks good. A few typos and such... maybe, I think.

File: src/components/home/Home.tsx (hunk 4 above)
Comment (Erin-Mounts) [reply src/components/home/Home.tsx]: :+1: (&mdash;)

Comment (Erin-Mounts) [reply src/components/home/Home.tsx]: that's the html entity \&mdash\; by the way

File: src/components/home/Partners.tsx (hunk 5 above)
Comment (Erin-Mounts) [reply src/components/home/Partners.tsx:30]: possibly. Will leave for discussion with the wider group.

**Pull request merged**